types/src/on_chain_config/consensus_config.rs

```diff
@@ -70,7 +70,7 @@ impl OnChainConsensusConfig {
 
     pub fn quorum_store_enabled(&self) -> bool {
         match &self {
-            OnChainConsensusConfig::V1(_config) => false,
+            OnChainConsensusConfig::V1(_config) => true,
```

Comment:
Revert before landing

```suggestion
            OnChainConsensusConfig::V1(_config) => false,
```